templates/components/faceted-search/facets/hierarchy-children.html

```diff
@@ -7,7 +7,7 @@
             rel="nofollow"
             data-id="{{ id }}"
             data-faceted-search-facet>
-                {{title}}
+                {{{title}}}
```

Comment:
We should probably `sanitize` all of the titles--there's little to no reason anyone should be using html in a custom field or option value name or value.

Reply:
added them.. pls review it.